Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -190,6 +196,8 @@ extension MessageManager: EngineWebDelegate {
 
     func routeChanged(newRoute: String) {
```

Comment:
[I asked the in-app squad how to get a callback when the "Show another message" button is clicked. They suggested using this function.](https://customerio.slack.com/archives/C036FTE6UJ0/p1720463076324169)